### DISCUSSION TITLE
[Request] Store TDM config next to .exe

### DISCUSSION THREAD

**Discussion by florianwgnr** [jziolkowski/tdm#97] [Ideas]
Hi, thanks for your work - this device manager is awesome!

Personally, i am using it as a "portable cloud app", which means i store the .exe in some kind of dropbox-folder, sync it and can start it from every PC i want.

But for now config files are stored here: C:\Users\<username>\TDM\tdm.cfg

My request is an option to store config files right next to the .exe, so i can sync one folder and got the .exe with all my configured settings.

**Comment by jziolkowski** [comment]
with the latest release you can use `--local` command-line parameter and it will store settings, devices and log file next to the executable
